### DISCUSSION TITLE
Stacked charging profiles

### DISCUSSION THREAD

**Discussion by muzammil-md92** [ChargeTimeEU/Java-OCA-OCPP#356] [Q&A]
What is the max number of stacked charging profiles for OCPP 1.6J and 2.0.1?

**Comment by Basler182** [comment]
Field type of stackLevel is `integer >= 0`.  The charging station provides you with its personal maximum value via its configuration `ChargeProfileMaxStackLevel`.

**Reply by muzammil-md92** [reply]
Thanks @Basler182. So it depend manufacturer to manufacturer how many they allowed to add on a single charger point.
